### PR TITLE
chore: Added kafkajs instrumentation and versioned tests skeleton

### LINKED ISSUE #2222: [kafkajs] Instrumentation baseline skeleton
### Description
In order to work in parallel on consumer and producer we should provide a baseline instrumentation that does the following:
 * instruments the `kafkajs` library and stub out a file to do instrumentation work in.
 * create a container that is running kafka that can be used for versioned tests

### PR DESCRIPTION
**Pull request by jsumners-nr**
This PR resolves #2222.

### CODE REVIEW

File: test/versioned/node-rdkafka/kafka.tap.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+tap.beforeEach(async (t) => {
+  t.context.agent = helper.instrumentMockedAgent()
+
+  const Kafka = require('node-rdkafka')
+  t.context.Kafka = Kafka
+
+  await new Promise((resolve) => {
+    const producer = new Kafka.Producer({
+      'metadata.broker.list': '127.0.0.1:9092'
```
Comment (bizob2828) [inline test/versioned/node-rdkafka/kafka.tap.js:20]: we put the host/port info in `test/lib/params.js`

File: test/versioned/node-rdkafka/kafka.tap.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+  await new Promise((resolve) => {
+    t.context.producer.disconnect(resolve)
+  })
+  await new Promise((resolve) => {
+    t.context.consumer.disconnect(resolve)
+  })
+})
+
+tap.test('stub', { timeout: 10_000 }, (t) => {
```
Comment (bizob2828) [inline test/versioned/node-rdkafka/kafka.tap.js:55]: I removed this timeout and it works as expected

File: test/versioned/node-rdkafka/package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "kafka-tests",
+  "targets": [{"name":"node-rdkafka","minAgentVersion":"11.18.0"}],
```
Comment (bizob2828) [inline test/versioned/node-rdkafka/package.json:3]: we'll have to tweak this once we release. we just released 11.18.0 so it's something greater than that

**Review by bizob2828 — COMMENTED** [review]
A few minor tweaks

File: test/versioned/node-rdkafka/kafka.tap.js (hunk 2 above)
Comment (jsumners-nr) [reply test/versioned/node-rdkafka/kafka.tap.js:55]: The timeout ensures that the test will get terminated and the cleanup actions will be performed when running the script directly (or other odd circumstances).

File: test/versioned/node-rdkafka/kafka.tap.js (hunk 1 above)
Comment (jsumners-nr) [reply test/versioned/node-rdkafka/kafka.tap.js:20]: Why? I see that the file introduces a bunch of environment variables for overriding things, but what utility are they providing?

**Review by bizob2828 — APPROVED** [review]

**Pull request merged**